src/Features/Onboarding.php

```diff
@@ -337,30 +337,38 @@ public static function get_allowed_industries() {
 	public static function get_allowed_product_types() {
 		$product_types = self::append_product_data(
 			array(
-				'physical'      => array(
+				'physical'        => array(
 					'label'       => __( 'Physical products', 'woocommerce-admin' ),
 					'description' => __( 'Products you ship to customers.', 'woocommerce-admin' ),
 				),
-				'downloads'     => array(
+				'downloads'       => array(
 					'label'       => __( 'Downloads', 'woocommerce-admin' ),
 					'description' => __( 'Virtual products that customers download.', 'woocommerce-admin' ),
 				),
-				'subscriptions' => array(
+				'subscriptions'   => array(
 					'label'   => __( 'Subscriptions', 'woocommerce-admin' ),
 					'product' => 27147,
 				),
-				'memberships'   => array(
+				'memberships'     => array(
 					'label'   => __( 'Memberships', 'woocommerce-admin' ),
 					'product' => 958589,
 				),
-				'composite'     => array(
+				'composite'       => array(
```

Comment:
This will probably require a rebase if you merge https://github.com/woocommerce/woocommerce-admin/pull/4703 first.

Reply:
yes, the joys of whitespace merging ;-)